content/datasets/i_sec_ids.md

```diff
@@ -0,0 +1,82 @@
+---
+title: I-SEC-IDS
+---
+
+- [Overview](#overview)
+- [Environment](#environment)
+- [Activity](#activity)
+- [Contained Data](#contained-data)
+- [Papers](#papers)
+- [Links](#links)
+- [Data Examples](#data-examples)
+
+| <!-- -->                 | <!-- -->                                                          |
+| ------------------------ | ----------------------------------------------------------------- |
+| **Network Data Source**  | NetFlows                                                          |
+| **Network Data Labeled** | Yes                                                               |
+| **Host Data Source**     | -                                                                 |
+| **Host Data Labeled**    | -                                                                 |
+|                          |                                                                   |
+| **Overall Setting**      | Single OS                                                         |
+| **OS Types**             | Windows 10                                                        |
+| **Number of Machines**   | 1                                                                 |
+| **Total Runtime**        | n/a                                                               |
+| **Year of Collection**   | 2021                                                              |
+| **Attack Categories**    | DoS, Scanning                                                     |
+| **User Emulation**       | n/a                                                               |
+|                          |                                                                   |
+| **Packed Size**          | 66 MB                                                             |
+| **Unpacked Size**        | -                                                                 |
+| **Download Link**        | [goto](https://gitlab.unige.ch/Benedetto.Serinelli/i-sec-dataset) |
+
+***
+
+### Overview
+The I-SEC-IDS dataset (exact meaning of the acronym unknown) consists of a small amount of network flows collected from a singular VM under attack.
+It does not feature user behavior or any other notable properties.
+
+### Environment
+The collection environment consists of one host running two virtual machines, victim and attacker, operating on Windows 10 and Kali Linux, respectively.
```

Comment:
So the number of machines is 2 and not 1, right?

Reply:
I also thought about that and I'm not 100% sure - the most consistent would probably be to write 1 (+1 Attacker). For datasets like [SR-BH 2020](https://fkie-cad.github.io/COMIDDS/content/datasets/sr_bh_2020/), this number reads 1, even though there are an unknown number of attackers (because it's a honeypot). In other cases, like with [ADFA-WD](https://fkie-cad.github.io/COMIDDS/content/datasets/adfa_wd/), there was no information about the attacker, so I also just wrote 1. What do you think?

Reply:
OK then let's leave it like it is.